parts/linux/cloud-init/artifacts/components.json

```diff
@@ -390,20 +390,16 @@
         "v1.27.13-2",
         "v1.27.14-1",
         "v1.27.15",
-        "v1.27.16",
         "v1.28.10-1",
         "v1.28.9",
         "v1.28.9-1",
         "v1.28.11",
-        "v1.28.12",
         "v1.29.4",
         "v1.29.4-1",
         "v1.29.5-1",
         "v1.29.6",
-        "v1.29.7",
         "v1.30.1-1",
-        "v1.30.2-hotfix.20240613",
-        "v1.30.3"
+        "v1.30.2-hotfix.20240613"
```

Comment:
I didn't change these, I think it got a little merked up in a rebase from master. But seems like everything changed is the current state of components.kson